pwiz_tools/Shared/zedgraph/ZedGraph/ZedGraphControl.cs

```diff
@@ -606,17 +607,22 @@ protected override void OnPaint( PaintEventArgs e )
 				base.OnPaint( e );
 
 				// Add a try/catch pair since the users of the control can't catch this one
-				try
-				{
-					_masterPane.Draw(e.Graphics);
-					if (_isZooming)
-					{
-						Rectangle rect = CalcScreenRect(_dragStartPt, _dragEndPt);
-						Pen p = new Pen(Color.Black) {DashStyle = DashStyle.Dash};
-						e.Graphics.DrawRectangle(p, rect);
-					}
-				}
-				catch { }
+                try
+                {
+                    _masterPane.Draw(e.Graphics);
+                    if (_isZooming)
+                    {
+                        Rectangle rect = CalcScreenRect(_dragStartPt, _dragEndPt);
+                        Pen p = new Pen(Color.Black) {DashStyle = DashStyle.Dash};
+                        e.Graphics.DrawRectangle(p, rect);
+                    }
+                }
+                catch (Exception exception)
+                {
+                    //Without this exceptions in the custom paint code are hard to debug. You just get
+                    //an emty graph without any visble error message.
```

Comment:
This comment has a lot of typos in it. Probably best to delete the comment: the code needs no explanation.

Reply:
Fixed. I wouldn't say that 2 typos is a lot.